app/controllers/api/v1/users/squads_controller.rb

```diff
@@ -1,8 +1,12 @@
 class Api::V1::Users::SquadsController < ApplicationController
   def index
-    user = User.find_by(params[:id])
-    squads = UserSquad.where(user_id: user.id)
-    render json: UserSquadSerializer.new(squads), status: 201
+    user = User.find_by(id: params[:user_id])
+    if !user.nil?
+      squads = UserSquad.where(user_id: user.id)
```

Comment:
If you have setup the `has_many` association in the model you can use `user.squads`